lib/dns.js

```diff
@@ -240,9 +243,17 @@ function onresolve(err, result, ttls) {
 }
 
 
-function resolver(bindingName) {
-  var binding = cares[bindingName];
+class Channel {
+  constructor() {
```

Comment:
this is optional, but it *might* be worthwhile allowing the servers to be set here as an optional argument.